includes/Products.php

```diff
@@ -710,8 +709,11 @@ public static function get_available_product_attributes( \WC_Product $product )
 	 */
 	public static function get_distinct_product_attributes( \WC_Product $product ) {
 
-		// TODO: implement
-		return [];
+		return [
+			self::get_product_color_attribute( $product ),
```

Comment:
we should probably filter this array or something, right? If one of these isn't set, it'll be an empty string in the array if I'm reading this correctly.